### PR TITLE
Add instructions for apply job

### PR DESCRIPTION
**Pull request by KiLiG-fcx**
1. Apply instruction field added in upsertjob.
2. Instruction visible to employer/admin, invisible to refugee

### CODE REVIEW

File: src/components/job/UpsertJob.jsx [hunk 1]
```diff
@@ -436,6 +437,18 @@ function UpsertJobCore({ update }) {
...
+            Instructions of applying for the job
+          </label>
+          <textarea
+            type="text"
+            rows="4"
+            className="textarea textarea-bordered w-full"
+            placeholder="Please specify the instructions of how to apply for this job."
+            {...register("instruction", { required: true })}
+          />
+        </div>
```
Comment (makersmelx) [inline src/components/job/UpsertJob.jsx:451]: @nehakumari2307 could you help to see if this way of expression works?
i don't get what the field is about.

**Review by makersmelx — CHANGES_REQUESTED** [review]

File: src/components/job/UpsertJob.jsx (hunk 1 above)
Comment (nehakumari2307) [reply src/components/job/UpsertJob.jsx:451]: This field is to provide employer a chance to specify how refugees should apply to their job posting as in email them their resume, or send their resume through online portal etc

File: src/components/job/UpsertJob.jsx [hunk 2]
```diff
@@ -436,6 +437,18 @@ function UpsertJobCore({ update }) {
             {...register("description", { required: true })}
           />
         </div>
+        <div className="flex flex-row items-center mb-4">
+          <label className="label flex basis-44" htmlFor="instruction">
+            Instructions of applying for the job
```
Comment (nehakumari2307) [inline src/components/job/UpsertJob.jsx:442]: Can you reword the label to "Job Application Instruction" and below the text field add a text which says "Provide instructions on how to apply for this job - email resume, upload resume to a portal or others".  Also remove the placeholder.

Comment (nehakumari2307) [reply src/components/job/UpsertJob.jsx:442]: Also, please do the same for the Job description and shift details placeholder. Remove them and add the text below the text field as info text

**Review by nehakumari2307 — APPROVED** [review]

**Pull request merged**